### PR TITLE
[DOCS] Update alerts

### PR DESCRIPTION
**Pull request by JessSaavedra**
# Description
- Update styles from alerts
- Update styles from navigation buttons

# Screenshots
## Alerts
Captura desde 2024-02-09 02-05-34
Captura desde 2024-02-09 02-05-51

## Pagination buttons
Captura desde 2024-02-08 01-18-07

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Review by SuperYeehee — APPROVED** [review]

**Review by SuperYeehee — APPROVED** [review]

File: docs/docusaurus/src/theme/Admonition/Types.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import DefaultAdmonitionTypes from '@theme-original/Admonition/Types';
+
+function CtaAdmonition(props) {
+    return (
+        <div>
+            <h3>{props.title}</h3>
+            <div>{props.children}</div>
+        </div>
```
Comment (josectobar) [inline docs/docusaurus/src/theme/Admonition/Types.js]: I think we can replace the container with a `section` tag instead.
```suggestion
        <section>
            <h3>{props.title}</h3>
            <div>{props.children}</div>
        </section>
```

File: docs/docusaurus/src/theme/Admonition/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  if (props.type === 'danger') {
+    return <Admonition {...props} icon={<DangerIcon/>} />;
+  }
+
+  if (props.type === 'cta') {
+    return <Admonition {...props} icon={<CtaIcon/>} />;
+  }
+  return (
+    <Admonition {...props} />
+  );
```
Comment (josectobar) [inline docs/docusaurus/src/theme/Admonition/index.js]: Can we use a switch instead? I think it'll be more appropriate considering the different cases.

```js
switch(type) {

case "cta":
 return  <Admonition {...props} icon={<CtaIcon/>} />;
....
....
etc
```

**Review by josectobar — CHANGES_REQUESTED** [review]
These alerts are looking great. I'm just leaving a few small suggestions. Thanks!

**Review by josectobar — APPROVED** [review]
Thank you for making these changes!

**Pull request merged**